pages/hugo.yml

```
@@ -42,13 +42,13 @@ jobs:
           submodules: recursive
       - name: Setup Pages
         id: pages
-        uses: actions/configure-pages@v1
+        uses: actions/configure-pages@v2
       - name: Build with Hugo
         run: |
           hugo \
             --environment production \
             --minify \
-            --baseURL ${{ steps.pages.outputs.base_url }}
+            --baseURL ${{ steps.pages.outputs.base_url }}/
```

Would adding quotes around this feel better? 🤷🏻‍♂️ 

```suggestion
            --baseURL "${{ steps.pages.outputs.base_url }}/"
```

Shouldn't be _necessary_ but I wouldn't be shocked if someone thought the trailing slash afterward was a typo with the unquoted style. 😅